clueless/settings.py

```diff
@@ -118,3 +117,5 @@
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
+
+from .settings_secret import *
```

Comment:
when i go to the files changed tab I see this: 
![image](https://cloud.githubusercontent.com/assets/20866481/19929446/ccc8db7c-a0d9-11e6-9e70-c7499c03883c.png)